hr_timesheet_third_party/views/res_partner_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_res_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.form (third_party)</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="groups_id" eval="[(4, ref('hr_timesheet.group_hr_timesheet_user'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button name="action_third_party_timesheet"
+                    type="object"
+                    class="oe_stat_button"
+                    icon="fa-clock-o">
+                    <field name="third_party_unit_amount_total" widget="statinfo" />
```

Comment:
When this field becomes `float`, I'm not sure how widget should be the more suitable for it

Reply:
In this type of buttons you must always have the `statinfo` widget and does not allow to add any more. 
I had made tests putting it in `time` format and it loses all the formatting.

Reply:
Understood